test/lib/mayaUsd/undo/testUsdUndoManager.py

```diff
@@ -106,6 +107,7 @@ def testNestedUsdUndoBlock(self):
         # expect to have 2 items on the undo queue
         self.assertEqual(cmds.undoInfo(q=True), nbCmds+2)
 
+    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() >= 2, 'For unknown reason this test is failing on UFE v1')
```

Comment:
Do we have a JIRA ticket to keep track of this?

Reply:
Not yet.